mep/people/views.py

```diff
@@ -149,3 +155,54 @@ def get_queryset(self):
             Q(country__name__icontains=self.q) |
             Q(address__person__name__icontains=self.q)
         ).order_by('name', 'city', 'street_address')
+
+
+class PersonMerge(FormView):
+    form_class = PersonMergeForm
+    template_name = 'people/merge_person.html'
+
+    def get_success_url(self):
+        return reverse('admin:people_person_changelist')
+
+    def get_form_kwargs(self):
+        form_kwargs = super(PersonMerge, self).get_form_kwargs()
+        form_kwargs['person_ids'] = self.person_ids
+        return form_kwargs
+
+    def get_initial(self):
+        # default to first person selected (?)
+        # _could_ add logic to select most complete record,
+        # but probably better for team members to choose
+        self.person_ids = [int(pid) for pid in self.request.GET.getlist('ids')]
```

Comment:
I believe this is not working as it stands (having tested it and gotten a ValueError). `.getlist()` assumes that the querystring would be like `?ids=366&ids=365`, whereas this gets a single element list with a string that's `366,365` for example.